package.json

```diff
@@ -545,6 +561,42 @@
                     "items": {"type": "string"},
                     "description": "Paths to look for TLA+ modules.",
                     "default": []
+                },
```

Comment:
This can be done in a follow-up PR, but do you think we can identify reasonable default values for most configuration options, so we don’t need to expose all of them to users?